diff --git a/src/tobii_pytracker/configs/custom_config.py b/src/tobii_pytracker/configs/custom_config.py
--- a/src/tobii_pytracker/configs/custom_config.py
+++ b/src/tobii_pytracker/configs/custom_config.py
@@ -12,7 +12,7 @@ class CustomConfig:
     def read_config(filename):
         if not os.path.exists(filename):
             raise FileNotFoundError(f"Configuration file '{filename}' not found.")
-        with open(filename, "r") as f:
+        with open(filename, "r", encoding="utf-8") as f:
             return yaml.safe_load(f)

diff --git a/src/tobii_pytracker/datasets/custom_dataset.py b/src/tobii_pytracker/datasets/custom_dataset.py
--- a/src/tobii_pytracker/datasets/custom_dataset.py
+++ b/src/tobii_pytracker/datasets/custom_dataset.py
@@ -90,7 +90,7 @@ class TextDataset(CustomDataset):
         label_col = self.text_cfg["label_column_name"]
         text_col = self.text_cfg["text_column_name"]
         self.classes = [str(c).lower() for c in df[label_col].unique()]
-        self.classes.append("none")
+        self.classes.append("nie wiem")
         self.data = [{"class": str(r[label_col]).lower(), "data": str(r[text_col]), "id": str(r[text_col])[:20].replace(" ","_")}
                      for _, r in df.iterrows()]  # add short id
